<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayBack_Tool</class>
 <widget class="QWidget" name="PlayBack_Tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>429</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>436</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>436</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MonitorDataPlay</string>
  </property>
  <widget class="UserNameCtrl" name="playback_ip_lineEdit">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>1</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">UserNameCtrl#playback_ip_lineEdit
{
	border-style: outset; 
	border-width: 0px; 
	background-color: rgb(255, 255, 255,0);

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="playback_table_lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>11</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MON_DATA_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="playback_datetime_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>138</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>8</hour>
     <minute>8</minute>
     <second>8</second>
     <year>8888</year>
     <month>8</month>
     <day>8</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="playback_curtime_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playback_connect_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>连 接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playback_play_Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>播 放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playback_stop_Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>停 止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playback_confirm_Button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>间隔时间(ms):</string>
   </property>
  </widget>
  <widget class="UserNameCtrl" name="playback_delay_lineEdit">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>69</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">UserNameCtrl#playback_delay_lineEdit
{
	border-style: outset; 
	border-width: 0px; 
	background-color: rgb(255, 255, 255,0);

}
</string>
   </property>
  </widget>
  <widget class="QSlider" name="playback_Slider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-3600</number>
   </property>
   <property name="maximum">
    <number>3600</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="playback_textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="playback_status_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>status:--------------------------------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>52</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据播放倍速:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="playSpeed_comboBox">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>49</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserNameCtrl</class>
   <extends>QLineEdit</extends>
   <header>mylineeditctrl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
